docs/configuration/ai-analysis.rst

```diff
@@ -299,36 +299,320 @@ Advanced - Customizing HolmesGPT
+                type: "<string>"
+                description: "<string>"
+                required: <true|false>
+
+Toolset Fields
+--------------
+
+.. list-table::
+   :widths: 20 10 60 10
+   :header-rows: 1
+
+   * - **Parameter**
+     - **Type**
+     - **Description**
+     - **Required**
+   * - ``enabled``
+     - boolean
+     - Indicates whether the toolset is enabled. Defaults to ``true``. If set to ``false``, the toolset will be disabled.
+     - No
+   * - ``name``
+     - string
+     - A unique identifier for the toolset. Used for informational purposes and logging.
+     - **Yes**
+   * - ``description``
+     - string
+     - A summary of the toolset's purpose. This description is visible to the LLM and helps it decide when to use the toolset for specific tasks.
+     - No
+   * - ``docs_url``
+     - string
+     - A URL pointing to documentation related to the toolset.
+     - No
+   * - ``icon_url``
+     - string
+     - A URL to an icon representing the toolset.
+     - No
+   * - ``tags``
+     - list
+     - Tags for categorizing toolsets, ``core`` will be used for all Holmes features (both cli's commands and chats in UI). The ``cluster`` tag is used for UI functionality, while ``cli`` is for for command-line specific tools. Default to ``[core,]``.
+     - No
+   * - ``installation_instructions``
+     - string
+     - Instructions on how to install prerequisites required by the toolset.
+     - No
+   * - ``variables``
+     - dictionary
+     - A set of key-value pairs defined by the user, not seen by LLM. These variables are typically used for configuring API keys, credentials, or other sensitive values that the LLM does not have visibility into. Values can reference environment variables using the $VARIABLE_NAME syntax.
+     - No
+   * - ``prerequisites``
+     - list
+     - A list of conditions that must be met for the toolset to be enabled. Prerequisites can include commands or environment variables, or both.
+     - No
+   * - ``additional_instructions``
+     - string
+     - Additional shell commands or processing instructions applied to the output of tools in this toolset.
+     - No
+   * - ``tools``
+     - list
+     - A list of tools defined within the toolset. Each tool is an object with its own set of fields.
+     - **Yes**
+
+  **Tool Fields**
+
+.. list-table::
+   :widths: 20 10 60 10
+   :header-rows: 1
+
+   * - **Parameter**
+     - **Type**
+     - **Description**
+     - **Required**
+   * - ``name``
+     - string
+     - A unique identifier for the tool within the toolset.
+     - **Yes**
+   * - ``description``
+     - string
+     - A brief description of the tool's purpose. Helps Holmes decide when to use this tool.
+     - **Yes**
+   * - ``command``
+     - string
+     - A shell command template that the tool will execute. Can include variables and parameters using Jinja2 syntax (``{{ variable_name }}``).
+     - At least one of ``command`` or ``script`` is required
```

Comment:
This isn't accurate. It's not *at least one of*. It's *either*
Please update below as well